library/std/canon.qs

```diff
@@ -432,8 +436,17 @@ namespace Microsoft.Quantum.Canon {
     }
 
     /// # Summary
-    /// Applies a unitary operation on the target,
-    /// controlled on a state specified by a given bit mask.
+    /// Applies `oracle` on `target` when `controlRegister`
+    /// is in the state specified by `bits`.
+    ///
+    /// # Description
+    /// Applies a unitary operation `oracle` on the `target`, controlled
+/// on a state specified by a given bit mask `bits`.
+/// Bit at `bits[i]` corresponds to qubit at `controlRegister[i]`.
```

Comment:
```suggestion
    /// The bit at `bits[i]` corresponds to qubit at `controlRegister[i]`.
```

Reply:
Updated. Somehow the previously committed suggestions didn't have proper indents so I'll update that too.